src/platform/backends/lxd/lxd_virtual_machine.h

```diff
@@ -59,11 +59,14 @@ class LXDVirtualMachine final : public VirtualMachine
     VMStatusMonitor* monitor;
     bool update_shutdown_status{true};
     multipass::optional<IPAddress> ip;
-    const QUrl base_url;
     NetworkAccessManager* manager;
+    const QUrl base_url;
```

Comment:
And here? Was it clang that reordered these? I fail to see a pattern :)

Reply:
See above :smiley: